src/doc/install_pyenv.md

```diff
@@ -0,0 +1,39 @@
+# Install Silkaj on a Python environment
+
+## Install Pyenv
+
+### Retrieve pyenv sources
+```bash
+git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+```
+
+### Handle shell modifications: point 2,3 and 4.
+- [Follow pyenv install documentation](https://github.com/pyenv/pyenv#installation)
+
+
+### Install latest Python version and create pyenv
+```bash
+pyenv install 3.6.0
+pyenv shell 3.6.0
+pyenv virtualenv silkaj-env
+```
+
+## Install Silkaj
+
+### Retrieve silkaj sources
+```bash
+git clone https://github.com/duniter/silkaj.git
+cd silkaj
+```
+
+### Install dependencies and store them on pyenv environement
+```bash
+pip install -r requirements.txt --upgrade
```

Comment:
On the virtual env, there is only one `python` version installed, so `pip` is good:
```bash
pyenv activate silkaj-env 
pyenv-virtualenv: prompt changing not working for fish.

pip --version
pip 9.0.1 from /home/moul/.pyenv/versions/3.6.0/envs/silkaj-env/lib/python3.6/site-packages (python 3.6)
```